realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -45,7 +45,7 @@
  * @see io.realm.Realm
  * @see io.realm.DynamicRealm
  */
-abstract class BaseRealm implements Closeable {
+public abstract class BaseRealm implements Closeable {
```

Comment:
Yeah... i am thinking hard... making it public make this https://github.com/realm/realm-java/issues/1594#issuecomment-194175373 be a problem .... although i was wrong before (`getRealm` won't be a problem if `BaseRealm` is not public).